qrcode/main.py

```diff
@@ -18,30 +21,55 @@
 from qrcode.image.base import BaseImage
 from qrcode.image.pure import PyPNGImage
 
+T = TypeVar("T")
+
 ModulesType = List[List[Optional[bool]]]
 # Cache modules generated just based on the QR Code version
 precomputed_qr_blanks: Dict[int, ModulesType] = {}
 
+InputData = Union[str, bytes, util.QRData]
+
+GenericImage = TypeVar("GenericImage", bound=BaseImage)
 
-def make(data=None, **kwargs):
-    qr = QRCode(**kwargs)
+DEFAULT_ERROR_CORRECTION = constants.ERROR_CORRECT_M
+DEFAULT_BOX_SIZE = 10
+DEFAULT_BORDER = 4
+
+
+def make(
+    data: InputData,
```

Comment:
I removed `None` as default argument because seems like .add_method doesn't expect None as value and will brake for `None` value